docs/rfd/002.md

```diff
@@ -0,0 +1,70 @@
+---
+authors: Markus Rudy (@burgerdev)
+state: discussion
+---
+
+# Objective
+
+Define a format for GLOME public keys at rest.
+
+# Background
+
+The GLOME protocol definition does not deal with key material handling, and the
+reference implementation only implements a very rudimentary storage format -
+32 raw octets. This causes a variety of problems, e.g. when transferring keys
+between hosts or when specifying server keys for *GLOME Login*.
+
+See also [google/glome#100](https://github.com/google/glome/issues/100).
+
+# Requirements
+
+* A GLOME public key at rest should be unambiguously identifiable as such.
+* Public keys should be printable.
+* Public keys should be easily exchanged over any medium, potentially analog.
+* The public key format should allow associating additional context with a key.
```

Comment:
That was part of the RFD before, but I removed it because we now have an INI parser. Per-key config should go into the key's INI section now (as soon as we support multiple key sections, that is :D).

Reply:
Removed this requirement because it's obsolete.